src/config.cpp

```diff
@@ -784,6 +784,13 @@ const config::attribute_value* config::get(config_key_type key) const
 	return i != values_.end() ? &i->second : nullptr;
 }
 
+const config::attribute_value& config::get_or(config_key_type key, config_key_type default_key) const
+{
+    check_valid();
```

Comment:
> error: use of undeclared identifier 'check_valid'

I don't get why it's deemed undefined. Is it because it's called by an ``inline`` function?

Reply:
Well. That wasn't it :|

Reply:
Isn't `check_valid()` the function that @gfgtdf deleted recently?

Reply:
Oh! That makes sense. I didn't have that commit pulled.